src/app/components/panel/panel.ts

```diff
@@ -50,6 +50,7 @@ import { PanelAfterToggleEvent, PanelBeforeToggleEvent } from './panel.interface
 
                         <ng-template *ngTemplateOutlet="headerIconTemplate; context: { $implicit: collapsed }"></ng-template>
                     </button>
+                    <button (click)="toggle($event)">collapse</button>
```

Comment:
I don't understand your PR abour multiselect component but you are modifying another components.